### ISSUE TITLE
SiteImprove reporting issue with main navigation keyboard accessibility

### ISSUE THREAD

**Issue by Stanton** [jadu/pulsar#1480]
**Describe the bug**
The current version of the SiteImprove accessibility checker reports an issue relating to the main navigation component. We are confident that the component's level of accessibility meets the guidelines, however due to the persistent nature of this component I recommend we take remedial action to prevent SI from reporting this issue, likely by the management of zero and negative tabindexes on hidden panels.

Failure to address this will result in repeated queries from QA teams and potentially start to be flagged by cypress.

Initially observed in Jadu Connect, verified in Pulsar Lexicon.

Jira Ticket: QU-5351

**Expected behavior**
The SiteImprove tool should no longer report this issue.

**Screenshots**
![Screenshot 2023-01-30 at 09 23 02](https://user-images.githubusercontent.com/18653/215503185-578bcb69-48ba-4b89-84af-1f98dfffa5ff.png)
